app/pages/search/controls/PositionControl.spec.js

```diff
@@ -0,0 +1,39 @@
+import { expect } from 'chai';
+import React from 'react';
+import Button from 'react-bootstrap/lib/Button';
+
+import { shallowWithIntl } from 'utils/testUtils';
+import PositionControl from './PositionControl';
+
+function getWrapper(props) {
+  const defaults = {
+    geolocated: false,
+    onPositionSwitch: () => null,
+  };
+  return shallowWithIntl(<PositionControl {...defaults} {...props} />);
+}
+
+describe('pages/search/controls/PositionControl', () => {
+  it('renders a div.app-PositionControl', () => {
+    const wrapper = getWrapper();
+    expect(wrapper.is('div.app-PositionControl')).to.be.true;
+  });
+
+  describe('Button', () => {
+    it('does not have active class if not geolocated', () => {
+      const button = getWrapper().find(Button);
```

Comment:
- [ ] Here it's better to explicitly have `getWrapper({ geolocated: false })...` so the test isn't relying on defaults in the getWrapper and break if the value there would for some reason change. Plus the reader can immediately see just from looking at this one test what props affect the active class.